### PR TITLE
Gradient widget

### PR DESCRIPTION
**Pull request by sigvef**
Depends on #16.

image

### CODE REVIEW

File: src/GradientNode.js [hunk 1]
```diff
@@ -31,9 +39,9 @@
...
 
     getOutput() {
-      return this.value;
+      return this.value.clone();
     }
   }
 
   TextureGen.GradientNode = GradientNode;
-})(TextureGen, jscolor);
+})(TextureGen, jscolor, jQuery);
```
Comment (stianjensen) [inline src/GradientNode.js]: Importing jscolor crashes for me

**Comment by sigvef** [comment]
ping

**Comment by stianjensen** [comment]
also, jscolor is still requested from index.html

**Comment by sigvef** [comment]
ping

**Comment by stianjensen** [comment]
I think you missed a line in the rebase, which was deleted by this prq earlier:
https://github.com/sigvef/texturegen/pull/17/files#diff-b3e21e07579c3a351a6d4c7e92b387beR44

**Comment by sigvef** [comment]
ping

**Pull request merged**